packages/story-editor/src/components/tabview/index.js

```diff
@@ -259,12 +260,13 @@ function TabView({
       {tabs.map(({ id, title, icon: Icon, ...tabRest }) => (
         <Tab
           key={id}
-          ref={(tabRef) => {
+          ref={(node) => {
             if (tabRefs) {
-              tabRefs[id].current = tabRef;
+              tabRefs[id].current = node;
             } else {
-              internalTabRefs.current[id] = tabRef;
+              internalTabRefs.current[id] = node;
             }
+            onTabRefUpdated(node, id);
```

Comment:
🤔 Wondering what the use case is for this - I don't think this is used anywhere, right? Maybe we should leave this out until we need it in the future?

Reply:
It's used in the follow-up PR #8523. :)